integration-cli/docker_api_build_test.go

```diff
@@ -53,27 +51,21 @@ COPY * /tmp/
 RUN find / -name ba*
 RUN find /tmp/`,
 	})
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 	defer server.Close()
 
 	res, body, err := sockRequestRaw("POST", "/build?dockerfile=baz&remote="+server.URL()+"/testD", nil, "application/json")
-	c.Assert(err, check.IsNil)
-	c.Assert(res.StatusCode, check.Equals, http.StatusOK)
+	c.Assert(err, checker.IsNil)
+	c.Assert(res.StatusCode, checker.Equals, http.StatusOK)
 
 	buf, err := readBody(body)
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 
 	// Make sure Dockerfile exists.
 	// Make sure 'baz' doesn't exist ANYWHERE despite being mentioned in the URL
 	out := string(buf)
-	if !strings.Contains(out, "/tmp/Dockerfile") ||
-		strings.Contains(out, "baz") {
-		c.Fatalf("Incorrect output: %s", out)
-	}
+	c.Assert(out, checker.Contains, "/tmp/Dockerfile")
+	c.Assert(out, checker.Not(checker.Contains), "baz")
```

Comment:
Thanks :blush: 

That said, it would be really easy to add, could just add `NotContains = checker.Not(shaker.Contains)` in `pkg/integration/checker/checker.go` :smile_cat: